portfolio/src/main/webapp/index.html

```diff
@@ -3,133 +3,143 @@
   <head>
     <meta charset="UTF-8">
     <title>Justine Ventura</title>
-    <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+    integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
   </head>
-  <header>
-      <!--style="color:darkslategrey;
-    font-family:Questrial;
-    font-size:85px;
-    padding-top:20px;
-    text-align=center;
-    padding-left:90px;"-->
-    <div class="center">
+  
+  <body> 
 
-       <h1 class="center" style="color:darkslategrey;margin:auto;width=50%;
-    font-family:Questrial;
-    font-size:85px;
-    padding-top:20px;
-    text-align=center;
-    padding-left:90px;">Justine Ventura</h1>
-   
-    </div>
+    <header>
+      <h1 class="center">Justine Ventura</h1>
       <img class="pfp ctr" style="text-align:center" src="img/bitmoji.png"/>
       <br/>
-      <div style="margin:auto;width=50%;text-align:center;padding-bottom:15px">
-           <a href="https://www.linkedin.com/in/justine-ventura-6275a8149?originalSubdomain=ca"><i class="fa fa-linkedin fa-3x" 
-      style="color:darkslategrey;
-    border-radius: 50%; margin-left:15px;margin-right:15px" aria-hidden="true"></i></a>
-      <a href="https://github.com/j-vent"><i class="fa fa-github fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px"  aria-hidden="true"></i></a>
-      <a href="https://moma.corp.google.com/person/justineventura"><i class="fa fa-google fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
-    <a href="mailto:justine.ventura14@gmail.com"><i class="fa fa-envelope fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
-    <a href="https://www.instagram.com/venturadventures/?hl=en"><i class="fa fa-instagram fa-3x" style="color:darkslategrey;
-    border-radius: 50%; margin-right:15px" aria-hidden="true"></i></a>
+      <div id="iconlist">
+        <a href="https://www.linkedin.com/in/justine-ventura-6275a8149?originalSubdomain=ca" 
+           class="icon fa fa-linkedin fa-3x"></a>
+        <a href="https://github.com/j-vent" class="icon fa fa-github fa-3x"></a>
+        <a href="https://moma.corp.google.com/person/justineventura" class="icon fa fa-google fa-3x"></a>
```

Comment:
probably shouldn't put moma links on a public web page. Be aware that these repositories are public.

Reply:
ah right... well it's on the commit history now unfortunately. sorry about that